test/test_other.py

```diff
@@ -2915,6 +2915,12 @@ def test_embind_tsgen(self):
     self.assertNotExists('out.js')
     self.assertFileContents(test_file('other/embind_tsgen.d.ts'), actual)
 
+  def test_embind_val_tsgen(self):
```

Comment:
Should this be called `test_embind_tsgen_val` so that you can run `test_embind_tsgen*` to run all the tsgen tests?

Reply:
Yeah, I always forget wildcard matches work.